content/guides/references/migration-guide.md

```diff
@@ -657,70 +669,46 @@ Here are a few examples, assuming the value of `videosFolder` is
   - `cypress/videos/file/one.cy.js.mp4`
   - `cypress/videos/two.cy.js.mp4`
 
-### Command / Cypress API changes
+### Command / Cypress API Changes
 
 #### `cy.mount()`
 
-A default `cy.mount()` command has been added. Note that this command only warns
-and redirect users to the API page, the user still needs to manually define
-their own `cy.mount()` command with `Cypress.Commands.add('mount', ...)`.
+A default `cy.mount()` command has been added for mounting components in
+Component Testing. Note that this command only warns and redirect users to the
+API page, the user still needs to manually define their own `cy.mount()` command
+with `Cypress.Commands.add('mount', ...)`.
+
+If you set up your app using the automatic configuration wizard, a basic
+`cy.mount` command will have been created in your support file. You can
```

Comment:
```suggestion
`cy.mount()` command will be imported for you in your support file from one our supported frameworks. You can
```
idk i don't love my suggestion, but this is only setting this up, not really creating something for users.